-- Bump cmake_minimum_required to 3.5 for CMake 4.0 compatibility.
-- CMake 4.0 removed support for cmake_minimum_required(VERSION < 3.5).
-- This template is configured into a nested CMake project that does not inherit
-- CMAKE_POLICY_VERSION_MINIMUM from the top-level build.

--- analyzer/python/settings.cmake.in.orig	2026-08-27 09:02:53 UTC
+++ analyzer/python/settings.cmake.in
@@ -40,7 +40,7 @@
 #
 ################################################################################

-cmake_minimum_required(VERSION 3.4.3 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.5 FATAL_ERROR)

 set(SRC_DIR "@CMAKE_CURRENT_SOURCE_DIR@")
 set(BIN_DIR "@CMAKE_CURRENT_BINARY_DIR@")
